### ISSUE TITLE
This bundle don't work with Symfony 3.4

### ISSUE THREAD

**Issue by brieucthomas** [php-http/HttplugBundle#238]
| Q            | A
| ------------ | ---
| Bug?         | yes
| New Feature? | no
| Version      | any

#### Actual Behavior

This bundle don't work with Symfony 3.4 (error occurred when the cache is clear) :

```
Fatal error: Cannot use Http\Client\Common\PluginClientFactory as PluginClientFactory because the name is already in use in vendor\php-http\httplug-bundle\Collector\PluginClientFactoryListener.php on line 5
```

#### Expected Behavior

Work with Symfony 3.4.

#### Steps to Reproduce

Clear cache with Symfony 3.4 and Httplug Bundle 1.*. 

#### Possible Solutions

There is already a class called ```PluginClientFactory``` in the ```Http\HttplugBundle\Collector``` namespace.

So, use another alias to ```Http\Client\Common\PluginClientFactory``` in the ```Http\HttplugBundle\Collector\PluginClientFactoryListener``` : 

```php
<?php

namespace Http\HttplugBundle\Collector;

// Before :
use Http\Client\Common\PluginClientFactory

// After
use Http\Client\Common\PluginClientFactory as CommonPluginClientFactory;
```

**Comment by fbourigault** [comment]
This was already [reported](https://github.com/php-http/HttplugBundle/issues/234) and is a false positive. Could you upgrade your PHP Version to 7.0.7+?

**Comment by brieucthomas** [comment]
Hello, 

Thanks for your response and sorry for the noise. 

But I cannot upgrade my PHP version (5.6.22). And if your bundle needs PHP 7.0.7+, update your compose.json.

**Comment by fbourigault** [comment]
This bundle should work with 5.6. However I don't see anything in the changelog pointing that a collision bug was fixed.

**Comment by Nyholm** [comment]
Our bundle does not need PHP7.0.7+. It is a bug in PHP which was fixed in 7.0.7. 

If you want to be at the latest version for HTTPlug bundle you should consider updating your environment to at least supported versions.

**Comment by brieucthomas** [comment]
Thank you @fbourigault !
